51.md

```diff
@@ -123,12 +123,14 @@ Then the user would create a 'Categorized Bookmarks' list event like below:
 
 ## List Event Kinds
 
-| kind   | list type               |
-| ------ | ----------------------- |
-| 10000  | Mute                    |
-| 10001  | Pin                     |
-| 30000  | Categorized People      |
-| 30001  | Categorized Bookmarks   |
+| kind   | list type                 |
+| ------ | ------------------------- |
+| 10000  | Mute                      |
+| 10001  | Pin                       |
+| 10030  | Custom Emoji List         |
```

Comment:
@SnowCait i think if we remove the references to the 10030 list this is probably mergeable? @pablof7z what do you think?

Reply:
@erskingardner Do you mean to use #492 instead of 10030? (I've not read #492.)

Reply:
No – I think that #492 should also just be a new `kind` listed on NIP-51 just like the categorized emoji list. There isn't a need for an entirely new NIP there. I've talked to a good few folks recently and the use of the 10000's lists is actually very low among clients. e.g. Most are using 30000 lists for Mute lists. I think the best move going forward is to try and normalize/maximize the usage of the kind numbers in the 30k-40k range and use the kinds + standard tags to build lists that have more context. That would mean...for example, 

* 30000 - categorized people
* 30001 - categorized bookmarks
* 30030 - categorized emoji 
* ect...

Reply:
What `d` tag are they using for mute lists?

Reply:
Why even have the kind numbers, then? If the lists will only be indexed by the `d` tag and there is no fundamental difference between them then they could just use the same kind number for "generic list".

Reply:
> What d tag are they using for mute lists?

Usually just "mute" or "Mute". 🤷‍♂️ 

> Why even have the kind numbers, then? If the lists will only be indexed by the d tag and there is no fundamental difference between them then they could just use the same kind number for "generic list".

So that you can search/filter for them. Without something that identifies them you'd need to fetch all lists from a user, loop through them, and do the filtering yourself based on the items in the lists.

Reply:
>the use of the 10000's lists is actually very low among clients.

That is client problem, not NIP one. Why does NIP follow regacy clients which doesn't follow NIPs?
[My client](https://nostter.vercel.app/) uses 10000, 10001 and 10030 in fact.

Reply:
> Usually just "mute" or "Mute". 🤷‍♂️

Well, if clients have not bothered to agree among themselves when doing a simple thing as a mute list or to follow the NIP, what makes you think they will from now on?

Honestly, if people don't want to be compatible with other apps or they don't care, there is nothing we can do.

I was expecting you to tell me most clients had agreed on some standard that deviated from the NIP, in which case I would say we should change the NIP to reflect that -- but also ask where are they having these discussions, because that is a very important part of Nostr. But it looks like they haven't agreed on anything and apparently do not care.

Reply:
> I've talked to a good few folks recently and the use of the 10000's lists is actually very low among clients. e.g. Most are using 30000 lists for Mute lists.

> Most are using 30000 lists for Mute lists.

How many is "good few folks"? This is wrong in any case, they should be using 10000 for mute lists as specified in NIP-51. I'm guilty of this for implementing NIP-51 proposal in snort before it was merged (sorry) and it seems other clients have copied the code without checking the NIP. As Snowcait says "that is client problem, not NIP one." NIP is very clear on the kind used for mute list.

Going back to the topic of the conversation:

> I've come around to the point that we shouldn't have a kind:10030 as well.

> I think the best move going forward is to try and normalize/maximize the usage of the kind numbers in the 30k-40k range and use the kinds + standard tags to build lists that have more context

The point of the 10030 event is to have a list of current emoji packs (30030 events by you or others) that clients can fetch for the user and show autocompletion. Is like your Telegram sticker collection, you can browse sticker packs created by others and add them to your collection. The expectation is that you'll have the same emoji packs available across all clients and since it will only be one per pubkey I chose a replaceable event.

If using two different kinds is the problem we could use a 30001 list with well-known "d" tag like "emoji" for the same purpose. For instance satellite.earth does this to store a list of communities you are "subscribed" to using kind 30001 and "d" tag "communities" (this is not in the communities NIP yet, not sure why).